### PR TITLE
Adding DeepEcho

### PR DESCRIPTION
**Pull request by amontanez24**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/deepecho/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+about:
+  home: https://github.com/sdv-dev/DeepEcho
+  summary: Mixed-type multivariate time series modeling with generative adversarial networks.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/rdt
```
Comment (amontanez24) [inline recipes/deepecho/meta.yaml:47]: I want to assign our team to multiple conda-forge projects. I saw this is the suggested way here (https://conda-forge.org/docs/user/faq.html#faq-teams), but I was wondering if there is a way to change the name of our team?

**Comment by amontanez24** [comment]
@conda-forge/help-python
This package depends on `pytorch`, which doesn't support windows on `conda-forge` (https://anaconda.org/conda-forge/pytorch), causing the tests to fail. I tried to switch to only installing it for non windows environments, and installing `pytorch-cpu` for windows instead by using selectors, but then I got the `noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.` lint error. I then tried making the package non noarch, but then I get the `Non noarch packages should have python requirement without any version constraints.` lint error.

Our package only runs on python 3.6-3.8 at the moment so we need version constraints. What is the best way to get around the `pytorch` windows issue?

**Comment by BastianZim** [comment]
There are two ways to solve the windows PyTorch issue:

1. Don't offer a windows version. This means keeping the package `noarch` and just ignoring the missing pytorch. When your package is then being downloaded the conda-solver will inform the user that a dependency is missing and fail. This is expected and acceptable. Once `pytorch` is available on Windows, the user will then be able to install the package.
2. Don't be `noarch` anymore. In that case the python version restrictions can be implemented using `skip`.

**Comment by amontanez24** [comment]
> There are two ways to solve the windows `PyTorch` issue:
> 
> 1. Don't offer a windows version. This means keeping the package `noarch` and just ignoring the missing `PyTorch` error. When your package is then being downloaded the conda-solver will inform the user that a dependency is missing and fail. This is expected and acceptable. Once `PyTorch` is available on Windows, the user will then be able to install the package.
> 2. Don't be `noarch` anymore. In that case, the python version restrictions can be implemented using `skip`: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skipping-builds and the selectors are here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
> 
> As a note though, python3.6 is not supported on conda-forge anymore, you're package will therefore only be built for `python3.7` and `python3.8`.
> 
> Option 1. is easier to implement and maintain but it means that no one on Windows will be able to install your package until `PyTorch` is available. If that is necessary, you'll have to go with option 2. I would also suggest following [conda-forge/pytorch-cpu-feedstock#32](https://github.com/conda-forge/pytorch-cpu-feedstock/issues/32) as that tracks windows support.

I think we'd like to go with option 1. Is there anything else that would need to be done to make that work?
The osx build failing is puzzling because it passed before and the error seems to have to do with the importing of a package called `jmp` (https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/402443/logs/25). 
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=402443&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=2585

I would just restart the workflow but I don't have permission.

We also had a question about using the same maintainers as another library we maintain. Are we good to just put `conda-forge/rdt` and is there anyway to change that name?

**Comment by BastianZim** [comment]
Before I have a look at the rest of the recipe:

> I would just restart the workflow but I don't have permission.

You can just close and reopen the PR.

> We also had a question about using the same maintainers as another library we maintain. Are we good to just put conda-forge/rdt and is there anyway to change that name?

Yeah I saw it but didn't answer that because I'm not 100% sure. :) 
These teams are created for a feedstock so the team will probably always have the name of the feedstock. Therefore, I'd say no but a reviewer from core might be able to answer that better. 
You're good though with just adding the conda-forge/rdt team as long as you follow https://conda-forge.org/docs/user/faq.html#faq-teams

File: recipes/deepecho/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - deepecho
+    - deepecho.models
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sdv-dev/DeepEcho
```
Comment (BastianZim) [inline recipes/deepecho/meta.yaml:40]:
```suggestion
  home: https://sdv.dev
```

File: recipes/deepecho/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  summary: Mixed-type multivariate time series modeling with generative adversarial networks.
+  license: MIT
+  license_file: LICENSE
```
Comment (BastianZim) [inline recipes/deepecho/meta.yaml:43]:
```suggestion
  license_file: LICENSE
  dev_url: https://github.com/sdv-dev/DeepEcho
  doc_url: https://sdv.dev/SDV
```

**Review by BastianZim — COMMENTED** [review]
I have added two small cosmetic comments. These aren't necessary and you'll have to double-check that I linked to the correct website but it might be nice to inform your users of your websites.

**Comment by BastianZim** [comment]
You can ignore the `jmp` error, that looks like a conda-forge bug.

File: recipes/deepecho/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - tqdm >=4.10,<5
+
+test:
+  imports:
+    - deepecho
+    - deepecho.models
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (BastianZim) [inline recipes/deepecho/meta.yaml:37]:
```suggestion
    - pip
    - pytest
  source_files:
    - tests
```

File: recipes/deepecho/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    - python >=3.6,<3.9
+    - pytorch >=1.4,<1.8
+    - tqdm >=4.10,<5
+
+test:
+  imports:
+    - deepecho
+    - deepecho.models
+  commands:
+    - pip check
```
Comment (BastianZim) [inline recipes/deepecho/meta.yaml:35]:
```suggestion
    - pip check
    - pytest
```

**Review by BastianZim — COMMENTED** [review]
Since you distribute your tests through PyPI, you can also add them here. This is not necessary but might make it easier to catch conda-forge related errors.

File: recipes/deepecho/meta.yaml (hunk 3 above)
Comment (amontanez24) [reply recipes/deepecho/meta.yaml:47]: Thanks! I think we will be updating the docs and possibly moving them around so we will probably leave this for now and add it in later.

**Comment by BastianZim** [comment]
If you’re otherwise ready, feel free to ping the review team as outlined in the PR description when you edit it (I‘m just a normal user so I can’t merge it).

**Comment by amontanez24** [comment]
@conda-forge/staged-recipes This Pr should be ready for review

**Comment by amontanez24** [comment]
@conda-forge-admin, please ping team
This PR is ready, it just fails because torch has not put up the latest version on conda-forge for windows

**Pull request merged**